### ISSUE TITLE
Module not found: Error: Can't resolve 'ui/core/view/view-common'

### ISSUE THREAD

**Issue by bellalMohamed** [mukaschultze/nativescript-windowed-modal#16]
### Which platform(s) does your issue occur on?
- Android
- Android versions: Not android version related
- Not device related

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.2.2
- Cross-platform modules: : ^6.2.1
- Runtime(s): ```"tns-android": {
      "version": "6.1.2"
    },
    "tns-ios": {
      "version": "6.1.0"
    }```

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

Just add the plugin to any project

### Is there any code involved? 

```
import { overrideModalViewMethod } from "nativescript-windowed-modal"
overrideModalViewMethod()
```

```
ERROR in ../node_modules/nativescript-windowed-modal/windowed-modal.js
Module not found: Error: Can't resolve 'ui/core/view/view-common' in '/Users/bellalmohamed/Development/zeal/zeal-alsagheer-app/node_modules/nativescript-windowed-modal'
 @ ../node_modules/nativescript-windowed-modal/windowed-modal.js 7:17-52
 @ ../node_modules/nativescript-windowed-modal/index.js
 @ ./app/components/navbar/navbar.component.ts
 @ ./app/components/navbar/navbar.module.ts
 @ ./app/app.module.ts
 @ ./main.ts
```

**Comment by growerp** [comment]
for me the problem is only on IOS, Android is OK
Demo works fine on IOS.

about 4 weeks ago it was working fine, now it is not....looks like some conflict with other packages?
any suggestions?
my package.json:
{
  "name": "growerp-restaurant",
  "version": "1.0.0",
  "description": "GrowERP for the restaurant and bar business.",
  "author": "AntWebsystems co.,Ltd",
  "license": "MIT",
  "nativescript": {
    "id": "com.growerp.restaurant",
    "tns-ios": {
      "version": "6.2.0"
    },
    "tns-android": {
      "version": "6.2.0"
    }
  },
  "dependencies": {
    "axios": "^0.19.0",
    "nativescript-accordion": "^6.0.0-beta.2",
    "nativescript-admob": "^4.0.0",
    "nativescript-appversion": "^1.4.2",
    "nativescript-bitmap-factory": "^1.8.1",
    "nativescript-camera": "^4.5.0",
    "nativescript-drop-down": "^5.0.4",
    "nativescript-exit": "^1.0.1",
    "nativescript-http-formdata": "^2.0.0",
    "nativescript-imagepicker": "^7.1.0",
    "nativescript-printer": "^1.4.0",
    "nativescript-purchase": "^2.0.13",
    "nativescript-socket.io": "^0.11.1",
    "nativescript-theme-core": "^1.0.4",
    "nativescript-toasty": "^2.0.1",
    "nativescript-ui-calendar": "^6.0.0",
    "nativescript-ui-chart": "^7.0.0",
    "nativescript-ui-dataform": "^6.0.0",
    "nativescript-ui-listview": "^8.0.1",
    "nativescript-ui-sidedrawer": "^8.0.0",
    "nativescript-vue": "^2.4.0",
    "nativescript-windowed-modal": "^6.0.0",
    "net": "^1.0.2",
    "source-map-support": "^0.5.10",
    "tns-core-modules": "^6.2.1",
    "typescript": "^3.2.2",
    "vue-i18n": "^8.8.0",
    "vuex": "^3.0.1"
  },
  "devDependencies": {
    "@babel/core": "~7.1.0",
    "@babel/preset-env": "~7.1.0",
    "babel-loader": "~8.0.0",
    "chai": "4.2.0",
    "clean-webpack-plugin": "^0.1.19",
    "css-loader": "^1.0.0",
    "karma": "4.3.0",
    "karma-chai": "0.1.0",
    "karma-mocha": "1.3.0",
    "karma-nativescript-launcher": "0.4.0",
    "karma-webpack": "3.0.5",
    "mocha": "6.2.0",
    "nativescript-dev-webpack": "~1.3.0",
    "nativescript-unit-test-runner": "^0.7.0",
    "nativescript-vue-template-compiler": "~2.4.0",
    "node-sass": "^4.7.1",
    "vue-loader": "~15.4.0",
    "webpack": "^4.16.4",
    "webpack-bundle-analyzer": "^3.4.1",
    "webpack-cli": "^3.3.9"
  }
}

**Comment by timdoege** [comment]
I get this error on both platforms for NS 6.2+ (the two platform specific files perform the same import).

**Comment by bellalMohamed** [comment]
Yes, Now I'm getting the error on both platforms

**Comment by mukaschultze** [comment]
Fixed on nativescript-windowed-modal@6.2.0

**Comment by growerp** [comment]
Thanks for the quick response and fix, working fine!
